cli/commands/profile/command.go

```diff
@@ -266,3 +264,10 @@ func CurrentCommand(builder Builder) *cobra.Command {
 
 	return cmd
 }
+
+func profileNameArgOrCurrent(ctx context.Context, args []string, profiler Profiler) (string, error) {
+	if currentFlag || len(args) == 0 {
+		return profiler.GetCurrentProfileName(ctx)
+	}
+	return args[0], nil
```

Comment:
I think we should set `default` profile if there is not a current.  I blew away my profiles and got this error after `bindplane profile set --username blah`

```
Error: failed to read current: open /Users/dsvanlani/.bindplane/profiles/current: no such file or directory
exit status 1
```


```suggestion
	if len(args) > 0 {
        return args[0], nil
    }

    currentProfile, err := profiler.GetCurrentProfileName(ctx)
    if err != nil {
        return "default", nil
    }

    return currentProfile, nil
```

of course then we'd have to instruct them to `bindplane profile use default`. So I could go either way.

Reply:
I took a slightly different approach because I didn't want `bindplane profile get` to use `default` if there was no current profile, but it now works to delete your profiles folder and run `bindplane profile set --port 8000`.